src/zoid/buttons/util.js

```diff
@@ -222,3 +223,55 @@ export function applePaySession() : ?ApplePaySessionConfigRequest {
         return undefined;
     }
 }
+
+export function getButtonSize(props : ButtonProps, container : string | HTMLElement | void) : string | void {
+    let containerWidth = 0;
```

Comment:
Can we add a check at the beginning of the function to exit early if `container` is undefined? Ex:

```js
export function getButtonSize(props : ButtonProps, container : string | HTMLElement | void) : string | void {
    if (!container) {
        return;
    }
}
```